### PR TITLE
docs: add change ref parameters to product edit

### PR DESCRIPTION
**Pull request by alexgarel**
OpenAPI documentation of parameters that goes to change_ref.

### CODE REVIEW

File: docs/api/ref/requestBodies/change_ref_properties.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    description: |
+      Name of the app providing the information
+  app_version:
+    type: string
+    description: |
+      Version of the app providing the information
+  app_uuid:
+    type: string
+    description: |
+      When an app use a single user to log its contributions,
```
Comment (teolemon) [inline docs/api/ref/requestBodies/change_ref_properties.yaml:21]:
```suggestion
      When an app uses a single user to log its contributions,
```

File: docs/api/ref/requestBodies/change_ref_properties.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      it might be interesting to know which user of the app is providing the information.
+      You can use this field to provide an identifier (eg: an sha1 of the username)
```
Comment (teolemon) [inline docs/api/ref/requestBodies/change_ref_properties.yaml:23]:
```suggestion
      You can use this field to provide an identifier (eg: an sha1 of the username) that's privacy preserving. Make sure that your salt is strong, perfectly random and secret
```

File: docs/api/ref/requestBodies/change_ref_properties.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  User-Agent:
+    type: string
+    description: |
+      It is required that you pass a specific User-Agent header when you do an API request.
+      But some times it's not possible to modify such header
+      (eg. request using javascript in a browser).
```
Comment (teolemon) [inline docs/api/ref/requestBodies/change_ref_properties.yaml:29]:
```suggestion
      (eg. request using JavaScript in a browser).
```

File: docs/api/ref/requestBodies/change_ref_properties.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      In such case, you can override it with this parameter.
```
Comment (teolemon) [inline docs/api/ref/requestBodies/change_ref_properties.yaml:30]:
```suggestion
      In such cases, you can override it with this parameter.
```

File: docs/api/ref/requestBodies/change_ref_properties.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+type: object
+description: |
+  Properties that goes in change ref
```
Comment (teolemon) [inline docs/api/ref/requestBodies/change_ref_properties.yaml:3]: unclear what change ref is

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**